.github/workflows/test-packages.yml

```diff
@@ -0,0 +1,32 @@
+name: Test Packages
```

Comment:
[note for reviewer]

The goal of this new GH action is to have a nightly test build of the packages of the current tip of main. That avoids having a late failure in the release process, which can be frustrating.

Reply:
Will the result of this workflow be visible to anywhere? I like the idea, though I wonder if we can even notice the failure.

Reply:
Its the same as the Enterprise nightly builds on the app repo (which also run daily) - I think the author of the last commit gets a notification (or possibly all maintainers?). Plus you can always check via the Actions UI on the repo.